src/Common/src/System/Security/Cryptography/PasswordBasedEncryption.cs

```diff
@@ -1063,5 +1063,15 @@ internal static int NormalizeIterationCount(uint iterationCount)
 
             return (int)iterationCount;
         }
+
+        internal static int NormalizeIterationCount(int iterationCount)
+        {
+            if (iterationCount <= 0 || iterationCount > IterationLimit)
+            {
+                throw new CryptographicException(SR.Argument_InvalidValue);
+            }
+
+            return (int)iterationCount;
```

Comment:
No need to cast here, obviously. This is just `int` version of `NormalizeIterationCount` few lines above. Eventually the other version should go away.